doc/nrf/ug_partition_manager.rst

```diff
@@ -279,3 +279,72 @@ CMake usage
 
       --slot-size $<TARGET_PROPERTY:partition_manager,MCUBOOT_SLOT_SIZE>
 
+.. _ug_pm_predefined:
+
+Predefined configuration
+========================
+Predefined configuration is a feature intended for deployed products which consists of multiple images where
+only a subset of the included images will be upgradeable through firmware update mechanisms.
+One example of this is a device which includes a non-upgradable first stage bootloader and an upgradeable application.
+In these use cases, the images to be upgraded must be linked to the same address as that which is deployed.
+This to avoid accidental overrides, and incorrect configurations.
+
+Predefined configuration allows the user to configure the **dynamic partition**.
+The dynamic partition consists of all memory adjacent to the "app" partition which is not occupied by a predefined partition.
+When Partition Manager is executed, it only operates on the dynamic partition, assuming that all other memory is reserved.
+Within the dynamic partition, it is allowed to define new partitions.
+The dynamic partition is re-sized by adding or removing partitions to the predefined configuration.
+
+.. _ug_pm_predefined_providing:
+
+Providing predefined configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The predefined configuration is provided through a YAML formatted configuration file.
+This file is similar to the regular :file:`pm.yml` configuration files, except
+that it also defines the start address and size of all partitions.
+The given configuration for a build can be found in :file:`${BUILD_DIR}/zephyr/include/generated/partitions.yml`.
```

Comment:
I believe it would be best to use a different filename for the current configuration and the predefined configuration.

Reply:
prj.partitions.yml ?

Reply:
Good idea, will fix